x-pack/plugins/siem/kibana.json

```diff
@@ -3,6 +3,8 @@
   "version": "8.0.0",
   "kibanaVersion": "kibana",
   "configPath": ["xpack", "siem"],
+  "requiredPlugins": ["actions", "alerting", "features", "licensing"],
+  "optionalPlugins": ["encryptedSavedObjects", "ml", "security", "spaces"],
```

Comment:
Why doesn't `case` need to be declared here? Do all the necessary references just happen in `x-pack/plugins/siem/server/plugin.ts`?

Reply:
Heh, I think it's not there because we have no references to the case plugin contract in our code (i.e. "I missed it"). I'll follow up with an audit of all of our `import`s, because I know that we pull in code from other plugins as well.

/cc @rudolf: If we're importing arbitrary code paths from another plugin it needs to be declared as a dependency, right?

Reply:
Importing code from another plugin will add that code to your plugin's bundle so you can safely call it. But it doesn't make that plugin run or guarantee that it's running in the background. Any exported code should be stateless and if that's the case then this isn't a problem.

When you start using another plugin's setup / start API's you need to declare a dependency.

Having said that, I couldn't see any imports or setup / start API's being used from Case, but might have missed it.